include/manager.h

```diff
@@ -90,21 +34,34 @@ void print_full_cap(uintcap_t);
  * Compartment
  ******************************************************************************/
 
-// Number of capabilities required to perform a transition
-#define COMP_RETURN_CAPS_COUNT 2
-
 // Compartment configuration file suffix
 extern const char* comp_config_suffix;
 
-// Capabilities required to transition back into the manager once compartment
-// execution has finished
-extern void* __capability comp_return_caps[COMP_RETURN_CAPS_COUNT];
+/* Struct representing configuration data for one entry point; this is just
+ * information that we expect to appear in the compartment, as given by its
+ * compartment configuration file
+ */
+struct ConfigEntryPoint
```

Comment:
Is `Config` the right name here? It feels more like `EntryPointDef` might be a better name?

Reply:
I would like to re-think this part, which would involve potentially changing more stuff. I *could* for now just change this struct name, but I would propose I add another item in #14 and leave it for another, more consistent PR focused on this entry point stuff?

Reply:
Adding to the issue and doing it later is fine with me.